### PR TITLE
Refactored text overflow in dropdown

### LINKED ISSUE #1966: [SDC] Refactor text overflow in dropdown
**Problem**
Overflow texts gets cut off in dropdown

**Describe the solution you'd like**
Text should overflow to the next line. EmCare team has solved this in this commit https://github.com/google/android-fhir/commit/17774dabd17dadfc5fbbef1a07e871c0dbbf46e3

cc @santosh-pingle @kunjan8794 @khyativyasargus @jingtang10

### PR DESCRIPTION
**Pull request by khyativyasargus**
**IMPORTANT: All PRs must be linked to an issue (except for extremely trivial and straightforward changes).**

Fixes #1966 

**Description**
Overflow texts gets cut off in dropdown with current single line textview. When text is multiline , it should overflow to the next line.

**Alternative(s) considered**
Have you considered any alternatives? And if so, why have you chosen the approach in this PR?

**Type**
Choose one: Bug Fix

**Screenshots (if applicable)**

**Checklist**
- [x] I have read and acknowledged the [Code of conduct](https://github.com/google/android-fhir/blob/master/docs/CODE_OF_CONDUCT.md).
- [x] I have read the [Contributing](https://github.com/google/android-fhir/wiki/Contributing) page.
- [x] I have signed the Google [Individual CLA](https://cla.developers.google.com/about/google-individual), or I am covered by my company's [Corporate CLA](https://cla.developers.google.com/about/google-corporate ).
- [x] I have discussed my proposed solution with code owners in the linked issue(s) and we have agreed upon the general approach.
- [x] I have run `./gradlew spotlessApply` and `./gradlew spotlessCheck` to check my code follows the style guide of this project.
- [x] I have run `./gradlew check` and `./gradlew connectedCheck` to test my changes locally.
- [x] I have built and run the demo app(s) to verify my change fixes the issue and/or does not break the demo app(s).

### CODE REVIEW

**Comment by khyativyasargus** [comment]
@santosh-pingle @shelaghm 

Before
image


After fixes
image

**Comment by shelaghm** [comment]
Thanks @khyativyasargus Looks good to me!

File: datacapture/src/main/res/layout/drop_down_list_item.xml [hunk 1]
```diff
@@ -11,14 +11,22 @@
...
+    android:orientation="vertical"
+>
+
+    <TextView
+        android:id="@+id/answer_option_textview"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:drawablePadding="@dimen/icon_drawable_padding"
+        android:singleLine="false"
+        android:ellipsize="end"
```
Comment (santosh-pingle) [inline datacapture/src/main/res/layout/drop_down_list_item.xml:27]: nit : Can you please remove this if not its not required ?

**Comment by santosh-pingle** [comment]
Thanks! @khyativyasargus 
LGTM

File: datacapture/src/main/res/layout/drop_down_list_item.xml (hunk 1 above)
Comment (khyativyasargus) [reply datacapture/src/main/res/layout/drop_down_list_item.xml]: @santosh-pingle Removed unnecessary attributes. Please have a look.

Comment (santosh-pingle) [reply datacapture/src/main/res/layout/drop_down_list_item.xml]: @khyativyasargus  Can we please not remove `android:drawablePadding="@dimen/icon_drawable_padding"` attribute, can you please confirm once.

I see this attribute was added in the pr #1749

**Pull request merged**